docs/fsharp/get-started/index.md

```diff
@@ -1,16 +1,27 @@
 ---
-title: Getting Started with F#
-description: Find out how to get started with the F# programming language in .NET.
-ms.date: 09/06/2016
+title: Get Started with F#
+description: Find out how to get started with the F# programming language.
+ms.date: 12/08/2018
 ---
-# Getting Started with F# #
+# Get Started with F# #
 
-There are multiple ways to get started with F#.  We have multiple articles written to provide a guide for each major way.  You can use the following table to help in making a decision.
+You can get started with F# on your machine or online.
+
+## Get started on your machine
+
+We have multiple guides on how to install and use F# for the first time on your machine.  You can use the following table to help in making a decision.
 
 | OS | Prefer Visual Studio | Prefer Visual Studio Code | Prefer a command line |
 | -- |------------------------|--------------------------|-----------------------------|-------------------------|
 | Windows | [Get started with Visual Studio](get-started-visual-studio.md) | [Get started with VSCode and Ionide](get-started-vscode.md) | [Get started with the .NET Core CLI](get-started-command-line.md) |
 | macOS | [Get started with VS for Mac](get-started-with-visual-studio-for-mac.md) | [Get started with VSCode and Ionide](get-started-vscode.md) | [Get started with the .NET Core CLI](get-started-command-line.md) |
 | Linux | N/A | [Get started with VSCode and Ionide](get-started-vscode.md) | [Get started with the .NET Core CLI](get-started-command-line.md) |
 
-In general, there is no specific way to get started which is better than the rest.  We recommend trying all ways to use F# on your machine to see what you like the best!
+In general, there is no specific that is better than the rest. We recommend trying all ways to use F# on your machine to see what you like the best!
+
+## Get started online
+
+If you'd rather not install F# and .NET on your machine, you can also get started with F# in the browser:
+
+* [Introduction to F# on Azure Notebooks](https://notebooks.azure.com/Microsoft/projects/2018-Intro-FSharp/html/Introduction%20to%20FSharp.ipynb) is a [Jupyter notebook](https://jupyter.org/) on the free Azure Notebooks service. Just sign in with an account and you can hack away directly in the browser!
```

Comment:
do you need to specify which kind of account?

Reply:
Doesn't need an Azure account, so I'll say a Microsoft account here